config/katello-answers.yaml

```diff
@@ -91,6 +91,8 @@ foreman_proxy::plugin::remote_execution::ssh: false
 foreman_proxy::plugin::salt: false
 foreman_proxy_content:
   proxy_pulp_yum_to_pulpcore: true
+  allowed_import_path: ['/var/lib/pulp/sync_imports', '/var/lib/pulp/imports']
+  allowed_export_path: ['/var/lib/pulp/exports']
```

Comment:
If these do not change the defaults as they are defined in puppet-foreman_proxy_content (which I suspect we would want that module to define the Katello defaults for those paths) you will not need to add them to the answers file here as those defaults will get picked up automatically.

Reply:
@ehelms Ok, removing.